package.json

```diff
@@ -13,10 +13,10 @@
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
     "express-rate-limit": "^6.10.0",
+    "nups-whatsapp-web.js": "^1.23.1-alpha.1",
```

Comment:
The project is based on official library. What's the reason you injecting your own library?

Reply:
I'm sorry, I was doing some testing and forgot that it was in the pull request branch.
I already returned the package.json to its original form